src/logger.js

```diff
@@ -0,0 +1,10 @@
+
+module.exports = {
```

Comment:
You aim at disabling log when testing, right? What about using an environment variable instead?

In `package.json`:
```json
{
  "scripts": {
    "test": "NODE_ENV=test mocha ...",
  }
}
```

In `src/logger.js`:
```js
const env = process.env.NODE_ENV || "development"

module.exports = {
  log: (msg) => {
    if (env === "test") {
      return;
    }

    console.log(msg);
  }
};
```

Reply:
good idea